make-maintenance-tarball.sh

```diff
@@ -7,17 +7,10 @@ reset=`tput sgr0`
 die() { set +v; echo "$red$*$reset" 1>&2 ; exit 1; }
 
 TAR_GZ='hubmap-portal-maintenance.tar.gz'
-OUTPUT_DIR='tmp/hubmap-portal-maintenance/'
-PUBLIC_DIR='../../context/app/static/js/maintenance/public'
 
 cd context && npm run build:maintain
-cd -
-
-
-mkdir -p "$OUTPUT_DIR"
-cd "$OUTPUT_DIR"
-rm -rf *
-cp -r "$PUBLIC_DIR" .
+cd app/static/js/maintenance/
 
 tar -czvf "$TAR_GZ" public
-echo "${green}Send $TAR_GZ to Bill.$reset" 
\ No newline at end of file
+mv "$TAR_GZ" public
+echo "${green}Send $TAR_GZ in context/app/static/js/maintenance/public to Bill.$reset" 
```

Comment:
```suggestion
echo "${green}Send $TAR_GZ in $PWD/public to Bill.$reset" 
# TODO: Add `aws s3 cp` to this script, and give Bill the URL.
```

Reply:
@mccalluc `CleanWebpackPlugin` should empty the public dir before each build.